plugins/mcreator-localization/lang/texts.properties

```diff
@@ -879,8 +881,50 @@ elementgui.command.on_command_executed=Command executed procedure trigger
 elementgui.fuel.properties=Fuel properties
+elementgui.music_disc.event_rightclicked_air=When right clicked in air (player loc.))
+elementgui.music_disc.event_crafted_smelted=When item is crafted/smelted
+elementgui.music_disc.event_rightclicked_block=When right clicked on block (hand loc.))
+elementgui.music_disc.event_entity_hitwith=When living entity is hit with item
+elementgui.music_disc.event_inventory=When item in inventory tick
+elementgui.music_disc.event_inhand=When item in hand tick
+elementgui.music_disc.event_stopped_using=On player stopped using
+elementgui.music_disc.event_swing=When entity swings item
+elementgui.music_disc.music_mono_tip=<html>Sound to play:<br><small>Music needs to be mono to properly work
+elementgui.music_disc.disc_description=Music disc description:
+elementgui.music_disc.has_glowing_effect=Has glowing effect?
+elementgui.music_disc.disc_description_tip=<html>Special information about the music disc:<br><small>Separate entries with comma, to use comma in description use \\
+elementgui.music_disc.disc_texture=Disc texture
+elementgui.music_disc.error_disc_needs_name=Music disc needs a name
+elementgui.music_disc.error_disc_needs_description=Music disc needs a description
+elementgui.music_disc.error_needs_sound=Select a sound
 modelement.painting=Painting
 modelement.painting.description=Painting is a picture that you can put on your walls using the Painting item
+elementgui.painting.import_painting=Click this to import painting icon
+elementgui.painting.painting_texture=Painting texture:
+elementgui.painting.painting_width=Painting width: 
+elementgui.painting.painting_height=Painting height: 
+elementgui.painting.error_painting_needs_texture=Painting needs to have texture
+elementgui.potion.is_instant=Is instant
+elementgui.potion.is_bad=Is bad
+elementgui.potion.is_benefitical=Is benefitical
+elementgui.potion.event_potion_applied=When potion started/applied
+elementgui.potion.event_potion_tick=On potion active tick
+elementgui.potion.event_potion_expires=When potion expires
+elementgui.potion.effet_displayname=Potion effect display name: 
+elementgui.potion.import_potion_icon=Click this to import potion icon
+elementgui.potion.color=Potion color: 
+elementgui.potion.icon=Potion icon:
+elementgui.potion.has_bottle=Register potion effect bottles: 
+elementgui.potion.displayname=Potion display name: 
+elementgui.potion.instant=<html>Is this potion instantly applied?<br><small>If this is checked, potion tick procedure will not be called
+elementgui.potion.bad=Is this potion bad for the player?
+elementgui.potion.benefitical=Is this potion benefitical to the player?
+elementgui.potion.render_status_inventory=Render active potion status in inventory:
+elementgui.potion.render_status_hud=Render active potion status in HUD:
+elementgui.potion.error_potion_needs_icon=Potion needs to have icon
+elementgui.potion.error_potion_needs_displayname=Potion needs a display name

```

Comment:
Please add an underscore between display and name (both here and when accessing it).

Reply:
Done!